tests/PHPStan/Type/BooleanTypeTest.php

```diff
@@ -0,0 +1,130 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type;
+
+use PHPStan\TrinaryLogic;
+use PHPStan\Type\Constant\ConstantBooleanType;
+use PHPStan\Type\Constant\ConstantIntegerType;
+
+class BooleanTypeTest extends \PHPStan\Testing\TestCase
+{
+
+	public function testAccepts(): void
+	{
+		$booleanType = new BooleanType();
+
+		$this->assertTrue($booleanType->accepts(new BooleanType(), true)->yes());
+		$this->assertTrue($booleanType->accepts(new ConstantBooleanType(true), true)->yes());
+		$this->assertTrue($booleanType->accepts(new NullType(), true)->no());
+		$this->assertTrue($booleanType->accepts(new MixedType(), true)->yes());
+		$this->assertTrue($booleanType->accepts(new FloatType(), true)->no());
+		$this->assertTrue($booleanType->accepts(new StringType(), true)->no());
```

Comment:
should be done the same way as other tests - e.g. dataAccepts + testAccepts

Reply:
Ah... right. I copied it from `IntegerTypeTest`, that's why it's like that. I can fix them both.